granian/server.py

```diff
@@ -95,6 +96,7 @@ def __init__(
         respawn_interval: float = 3.5,
         reload: bool = False,
         process_name: Optional[str] = None,
+        pid: Optional[Path] = None,
```

Comment:
Same for the CLI parameter, I'd change this to `pid_file`

Reply:
Addressed via https://github.com/emmett-framework/granian/compare/405ff32e5fbe32a1ac6db956e6d9df2c71913be8..ea4efa7dad1e4b1fbe09d2dfd822936be9d2095d